splash/qtrender_lua.py

```
@@ -343,10 +344,26 @@ def error(error_info):
         ))
 
     @command(async=True)
-    def go(self, url, baseurl=None, headers=None):
+    def go(self, url, baseurl=None, headers=None, http_method="GET", body=None, formdata=None):
+
         if url is None:
             raise ScriptError("'url' is required for splash:go")
 
+        if http_method not in OPERATION_QT_CONSTANTS:
+            raise ScriptError("incorrect HTTP method: {}".format(http_method))
+
+        if formdata:
+            body = self.lua.lua2python(formdata)
+            if isinstance(body, dict):
+                body = urlencode(body)
+            else:
+                raise ScriptError("formdata argument for go() must be Lua table")
+
+        elif body:
+            body = self.lua.lua2python(body)
+            if not isinstance(body, basestring):
+                raise ScriptError("request body must be string")
```

I think we should raise an error if both formdata and body are passed